### ISSUE TITLE
Add an option for user to choose between UTC timecode and local time

### ISSUE THREAD

**Issue by muziqaz** [FoldingAtHome/fah-web-client-bastet#217]
I'm hearing more and more requests and disappointed comments about UTC being used in logs.
This is not mission critical, but it would be of the benefit for users to have this option, that they can easier troubleshoot any problems happening with WUs.
Some said, that if logs were in local time, they could much easier compare the timecodes with any other happenings in their network or around the house.

**Comment by Wedge009** [comment]
I disagree with logs being recorded in local time, which is so variable. But if there was a way to record times in UTC while displaying in the user's local time, I wouldn't object to that.

**Comment by jcoffland** [comment]
Yeah, it might be more convenient for some users if the logs were in local time but it would only cause more problems with trying to diagnose problems.  The logs are mainly for us not the users.
